### PR TITLE
Adding Power support(ppc64le) with continuous integration/testing to the project for architecture independent

### PR DESCRIPTION
**Pull request by asellappen**
Thanks for taking care of issue https://github.com/valyala/fasthttp/issues/894.

I have modified .travis.yml to support the package for power on ppc64le and comepleted the jobs,Can you please review and merge the change.

This helps us simplify testing later when distributions are re-building and re-releasing, For more info tag @gerrith3.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -59,3 +62,13 @@ jobs:
         - 1.15.x
       script:
         - go test -race -v ./...
+#Adding power on support
```
Comment (erikdubbelboer) [inline .travis.yml:65]:
```suggestion
```

**Review by erikdubbelboer — CHANGES_REQUESTED** [review]
Thanks. Please remove that one line that doesn't make sense and I'll merge it.

**Comment by asellappen** [comment]
removed comment as expected ,pls check and merge

**Comment by erikdubbelboer** [comment]
Thanks!

**Pull request merged**